src/test/java/co/omise/staging/StagingAccountTest.java

```diff
@@ -18,7 +19,8 @@ public void getAccount_success() throws Exception {
         Client client = new Client(STAGING_PKEY, STAGING_SKEY);
 
         // When
-        Account actualAccount = client.account().get();
+        Request<Account> getAccountRequest =  Account.buildGetRequest().build();
+        Account actualAccount = client.sendRequest(getAccountRequest, Account.class);
```

Comment:
Passing `Account.class` look a bit weird. Could we use generic type instead of passing class type?

Reply:
We've both looked into this and so far nothing has come to mind. It would be great if we can find a solution to this.

Reply:
Nice 👍